### ISSUE TITLE
FTP transfer

### ISSUE THREAD

**Issue by Invictaz** [bucanero/apollo-ps2#12]
Can you transfer savefiles from ps2 to ps3 and vice versa via a crosslink cable?

I mean .vme files from ps2 classics on ps3 that go to real ps2 and then decrypt it to use on real ps2?

**Comment by bucanero** [comment]
No, there's no network support on apollo-ps2. Keep in mind you can already decrypt VME directly on apollo-ps3.

Additional support for virtual memory cards on Apollo is under development.

**Comment by Invictaz** [comment]
Copying directly from PS2 to PS3 would be awesome.
